processor/logstransformprocessor/README.md

```diff
@@ -0,0 +1,24 @@
+# Logs Transform Processor
+
```

Comment:
Based on conversation in this week's collector SIG, this component will have to be clearly marked as experimental.